cli/server.go

```diff
@@ -297,6 +299,7 @@ func server() *cobra.Command {
 				TURNServer:           turnServer,
 				TracerProvider:       tracerProvider,
 				Telemetry:            telemetry.NewNoop(),
+				//LicenseHandler:       coderd.Licenses(),
```

Comment:
Was this accidentally commented out, or is the comment just there to illustrate the default value that's assigned in `coderd.New`?